packages/spark-core/base/typography.js

```diff
@@ -0,0 +1,28 @@
+/* global document Promise */
+const FontFaceObserver = require('fontfaceobserver');
+
+const typeLoader = ({ bodyFontName, headingFontName } = {}) => {
+  let bodyFont;
+  let headingFont;
+
+  if (bodyFontName) {
```

Comment:
We'll need to provide a useful error message if this fails.

Reply:
I think the useful error message issue is project-wide. I want to leave it as is until we decide how to handle it.

Reply:
It is is project-wide, but we gotta start somewhere.

Reply:
There's a lot of decisions to be made before we do error handling. I think it'd be wiser to create an issue to work it out before implementing.

Reply:
Created an issue to get it started.
https://github.com/sparkdesignsystem/spark-design-system/issues/687